code/README.md

```diff
@@ -1,16 +1,18 @@
 #### User-contributed code and links (![](/_res/l.png) icon) to third party resources ####
 
+**Active and prolific authors**
```

Comment:
Mind removing the "Active and " here? Would still be a useful link to keep even if he is no longer active someday, and if it gets too out of date can be removed.

Reply:
np :)

Reply:
Thank you!